payjoin-cli/Cargo.toml

```diff
@@ -37,7 +37,7 @@ hyper = { version = "1", features = ["http1", "server"], optional = true }
 hyper-rustls = { version = "0.26", optional = true }
 hyper-util = { version = "0.1", optional = true }
 log = "0.4.7"
-payjoin = { version = "0.23.0", default-features = false }
+payjoin = { path="../payjoin", default-features = false }
```

Comment:
This change doesn't need to be here. We're already patching `payjoin` from crates.io with our local path in the top level `Cargo.toml`

https://github.com/payjoin/rust-payjoin/blob/e2b66a2b11b8e41917afb5ca6d8abb749e5c9d43/Cargo.toml#L5-L6